index.html

```diff
@@ -4,8 +4,82 @@
      <meta charset="UTF-8">
      <meta name="viewport" content="width=device-width, initial-scale=1.0">
      <title>project 3</title>
+     <link rel="stylesheet" href="style.css">
+   <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Kanit:wght@300&display=swap" rel="stylesheet">
  </head>
  <body>
-     
+     <header>
+       <h1>
+        Design conference<br>
```

Comment:
This `br` is being used for a line break that is aesthetic / a design choice vs one that is meaningful, so you should instead handle this via CSS. You can do this by wrapping a different element around your text, like a `div`:

```suggestion
        <div>Design conference</div>
```